packages/synapse-react-client/src/components/SynapseTable/RowSelection/RowSelectionControls.tsx

```diff
@@ -66,7 +70,8 @@ export function RowSelectionControls(props: RowSelectionControlsProps) {
                 key={customControl.buttonText}
                 variant="contained"
                 callbackData={{
-                  data,
+                  tableId: entityId,
+                  queryMetadata: queryMetadata,
```

Comment:
Update callbackData to include `tableId` and only include `queryMetadata`, which specifically omits the row data. The implementations of these callbacks did not use the row data today, which would have only included one page of row data anyways, which isn't very useful.